adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java

```diff
@@ -50,6 +53,8 @@
 
     // Start of user code class_attributes
 	private static final Logger log = LoggerFactory.getLogger(RMToolManager.class);
+    private static Map<String, Requirement> requirements = new HashMap<String, Requirement>(1000);
+    private static int nextRequirementId;
```

Comment:
what's this? I guess it's a counter for the `trs:order`. In that case, it also has to be an AtomicLong, because otherwise two CEs may get the same order no.

Reply:
No no! This is not relating to TRS. requirements and nextRequirementId are simulating the back-end tool.